default-menu-auto-hide-fedora.ks.in

```diff
@@ -0,0 +1,17 @@
+#version=DEVEL
+#test name: default-menu-auto-hide
+%ksappend repos/default.ks
+
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+%post --nochroot
+SYSROOT=/mnt/sysroot
+
+grep -q "menu_auto_hide=1" $SYSROOT/boot/grub2/grubenv || \
```

Comment:
Could you please print the `grubenv` file? It helps with debugging.

Reply:
Fixed.